src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
```

Comment:
missing header comments!

when you create a new class, you need to let users and developers know what is its purpose.
hence, need a crisp header comment